### DISCUSSION TITLE
Enter and Leave events aren't fired

### DISCUSSION THREAD

**Discussion by Banbury** [Textualize/textual#4815] [Q&A] [answered]
I have added event handlers for Enter and Leave on a widget derived from Static. The widget has several children. The event handlers are never fired. Since the documentation says, that the event don't bubble, does that mean, that the child elements eat the events?
How else can I find out, if the mouse is over the widget and when it leaves?

**Comment by TomJGooding** [comment, marked as answer]
Sorry I spotted this was still unanswered, in case you missed it there was an update to Enter and Leave so they now bubble in #4818.

**Comment by Banbury** [comment]
I didn't miss it, but thanks anyway.
